resource/event_log.go

```diff
@@ -0,0 +1,840 @@
+}
+
+//PfrEventDataHeader structure represents PFR_EVENT_DATA_HEADER
+type PfrEventDataHeader struct { // Description
+	Version    uint8  // Version# of PFR_EVENT_DATA structure
+	Cpld       uint8  // CPLD# (0-based)
+	EventID    uint8  // Event Identifier
+	Attribute  uint8  // Extend and String Type information
+	Reserved   uint32 // Reserved for future use (set to 0)
+	InfoSize   uint32 // P, Size of event information in bytes
+	StringSize uint32 // S, Size of event string in bytes
+}
+
+//EventNameList - define map for event name
+var eventNameList = map[uint32]string{
+	0x00000000: "EV_PREBOOT_CERT",
+	0x00000001: "EV_POST_CODE",
+	0x00000002: "EV_UNUSED",
+	0x00000003: "EV_NO_ACTION",
+	0x00000004: "EV_SEPARATOR",
+	0x00000005: "EV_ACTION",
+	0x00000006: "EV_EVENT_TAG",
+	0x00000007: "EV_S_CRTM_CONTENTS",
+	0x00000008: "EV_S_CRTM_VERSION",
+	0x00000009: "EV_CPU_MICROCODE",
+	0x0000000A: "EV_PLATFORM_CONFIG_FLAGS",
+	0x0000000B: "EV_TABLE_OF_DEVICES",
+	0x0000000C: "EV_COMPACT_HASH",
+	0x0000000D: "EV_IPL",
+	0x0000000E: "EV_IPL_PARTITION_DATA",
+	0x0000000F: "EV_NONHOST_CODE",
+	0x00000010: "EV_NONHOST_CONFIG",
+	0x00000011: "EV_NONHOST_INFO",
+	0x00000012: "EV_OMIT_BOOT_DEVICE_EVENTS",
+	0x80000000: "EV_EFI_EVENT_BASE",
+	0x80000001: "EV_EFI_VARIABLE_DRIVER_CONFIG",
+	0x80000002: "EV_EFI_VARIABLE_BOOT",
+	0x80000003: "EV_EFI_BOOT_SERVICES_APPLICATION",
+	0x80000004: "EV_EFI_BOOT_SERVICES_DRIVER",
+	0x80000005: "EV_EFI_RUNTIME_SERVICES_DRIVER",
+	0x80000006: "EV_EFI_GPT_EVENT",
+	0x80000007: "EV_EFI_ACTION",
+	0x80000008: "EV_EFI_PLATFORM_FIRMWARE_BLOB",
+	0x80000009: "EV_EFI_HANDOFF_TABLES",
+	0x8000000A: "EV_EFI_PLATFORM_FIRMWARE_BLOB2",
+	0x8000000B: "EV_EFI_HANDOFF_TABLES2",
+	0x8000000C: "EV_EFI_VARIABLE_BOOT2",
+	0x80000010: "EV_EFI_HCRTM_EVENT",
+	0x800000E0: "EV_EFI_VARIABLE_AUTHORITY",
+	0x800000E1: "EV_EFI_SPDM_FIRMWARE_BLOB",
+	0x800000E2: "EV_EFI_SPDM_FIRMWARE_CONFIG",
+	0x401:      "PCR_MAPPING",
+	0x402:      "HASH_START",
+	0x403:      "COMBINED_HASH",
+	0x404:      "MLE_HASH",
+	0x40a:      "BIOSAC_REG_DATA",
+	0x40b:      "CPU_SCRTM_STAT",
+	0x40c:      "LCP_CONTROL_HASH",
+	0x40d:      "ELEMENTS_HASH",
+	0x40e:      "STM_HASH",
+	0x40f:      "OSSINITDATA_CAP_HASH",
+	0x410:      "SINIT_PUBKEY_HASH",
+	0x411:      "LCP_HASH",
+	0x412:      "LCP_DETAILS_HASH",
+	0x413:      "LCP_AUTHORITIES_HASH",
+	0x414:      "NV_INFO_HASH",
+	0x416:      "EVTYPE_KM_HASH",
+	0x417:      "EVTYPE_BPM_HASH",
+	0x418:      "EVTYPE_KM_INFO_HASH",
+	0x419:      "EVTYPE_BPM_INFO_HASH",
+	0x41a:      "EVTYPE_BOOT_POL_HASH",
+	0x4ff:      "CAP_VALUE",
+}
+
+var eventLogInfo *EventLogInfo
+
+//InitializeEventLogInfo - Function to create and intialize eventLogInfo variable
+func InitializeEventLogInfo() *EventLogInfo {
+	log.Trace("resource/event_log:InitializeEventLogInfo() Entering")
+	defer log.Trace("resource/event_log:InitializeEventLogInfo() Leaving")
+	eventLogInfo := new(EventLogInfo)
```

Comment:
EventLogInfo{} would set all the default values

Reply:
Resolving this comment as the code no longer exists.